ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml

```diff
@@ -80,7 +80,20 @@ tests:
     test:
     - as: test
       cli: latest
-      commands: JUNIT_DIRECTORY=${ARTIFACT_DIR} make e2e/olm
+      commands: JUNIT_DIRECTORY=${ARTIFACT_DIR} make e2e/olm SKIP='\[FLAKE\]'
```

Comment:
Note: this is just a stopgap until https://github.com/openshift/operator-framework-olm/pull/264 lands in the downstream OLM repository, which includes the upstream commit for segmenting flaky tests into their own github actions. We can look into consolidating this into a single target where the default behavior is to skip the flaky tests to avoid having to make changes to multiple repositories when we make a chance to any of the root/staging Makefiles.